common/changes/@uifabric/fabric-website-resources/codepen_2018-08-13-18-24.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
+      "packageName": "@uifabric/fabric-website-resources",
+      "type": "none"
```

Comment:
What effect does `none` have on publishing? I'm worried it won't publish a new version if this is the only change.. I think this should get out to users who may have been doing example dev.

Reply:
npm start experience does not depend on the version that's published... the npm install step already builds out the updated oufr for the local iterative dev experience.